docs/configuration/optional-settings.md

```diff
@@ -109,6 +109,16 @@ In order to send email, NetBox needs an email server configured. The following i
 * TIMEOUT - Amount of time to wait for a connection (seconds)
 * FROM_EMAIL - Sender address for emails sent by NetBox
 
+Email is sent from NetBox only for critical events. If you would like to test the email server configuration please use the following:
+
+```
+# python ./manage.py nbshell
+### NetBox interactive shell (jstretch-workstation)
```

Comment:
You can omit the header lines from the output to keep it a bit cleaner.

Reply:
Removed the headers